src/vendor/perf-goggles.js

```diff
@@ -115,7 +114,7 @@ const transformOptionCollectionHash = optionCollectionHash => {
 };
 
 const treeherderOptions = async () => {
-  const optionCollectionHash = await queryAllTreeherderOptions();
+  const optionCollectionHash = await queryAllTreeherderOptions;
```

Comment:
(why) What's the semantic difference here?  Callable vs. not?

Reply:
I have changed the definition of `queryAllTreeherderOptions`.  Before, it was an asynch function that you call to get Treeherder information. Now, it is a Promise to get Treeherder information.  The latter would make a real network call to get the data every time you called it.  The latter makes the call once (at creation time) and caches the result in itself for other promise chains to use. 

```
const promise = (asynch()=>{ /*do some network call here*/  })(); //only run once 
const result = await promise;  // use it anytime, and as many times as you like

Reply:
Ah!  Today I learned that Promises act as memoized procedures as well as bits of async code.  That makes things much clearer; thank you.